docs/ios.md

```diff
@@ -29,34 +29,42 @@ git clone --recursive https://github.com/kinfoundation/kin-core-ios
 
 This is how we did the Sample App - you might look at the setup for a concrete example.
 
-## API Usage
+## Getting Started
 
-The SDK exposes two classes, `KinClient`, and `KinAccount`.
+### Connecting to the Network
 
-### KinClient
-
-`KinClient` stores the configuration for the network, and is responsible for managing accounts.
+Create an instance of `KinClient`, providing the URL of the network entry-point, and the ID of the network.
 
 ```swift
-let kinClient = try? KinClient(with: URL, networkId: NetworkId)
+let url = URL(string: "https://horizon-kik.kininfrastructure.com")
+let kinClient = try? KinClient(with: url!, networkId: .mainNet)
 ```
 
-#### Account Management
+### Creating and retrieving instances of `KinAccount`
+
+On first use, an account must be created to send and receive KIN.  Account details are stored in the Keychain.  Multiple accounts may be created.
 
 ```swift
-public func addAccount() throws -> KinAccount
+let account = try kinClient.accounts.first ?? kinClient.addAccount()
+```
 
-public func deleteAccount(at index: Int) throws
+To retrieve a specific account:
 
-public private(set) var accounts: KinAccounts
+
+```swift
+let account = kinClient.accounts[1]
 ```
 
----
+An account may be deleted.  **WARNING:** deleting an account will prevent access to any KIN the account holds.  Deleting an account does **not** prevent the account from receiving additional KIN.
+
+```swift
+kinClient.deleteAccount(at: 0)
+```
 
-### KinAccount
+## Onboarding
 
 Before an account can be used on the configured network, it must be funded with the native network currency.
-This step must be performed by a service, and is outside the scope of this SDK.
```

Comment:
Can you add a word or 2 for the onboarding guide at [Onboarding](onboarding.md), also provide a complete example for the onboarding (against kin faucet and native faucet like in sample)

Reply:
I'll add the same text you have in the Android doc -- a pointer to the sample app.